modules/lagoon/lagoon_extras/README.md

```diff
@@ -0,0 +1,11 @@
+# GovCMS Platform
+
+Provides additional tooling and management processes for sites hosted on the govCMS platform. The module provides a number of service decorators to improve our ability to manage the platform.
+
+## Verbose logging
+
+```
+$config['lagoon_extras.settings']['verbose_logging']= getenv('GOVCMS_VERBOSE_LOGGING') ?: FALSE;
```

Comment:
This is currently missing from settings file, so we have no way to enable via env var at the moment